Cargo.lock

```diff
@@ -331,6 +331,7 @@ name = "gateway-api"
 version = "0.15.0"
 dependencies = [
  "anyhow",
+ "cfg-if",
```

Comment:
We don't have a series of cases, just the one, so at this point it seems like we can remove this new dependency:

```suggestion
```

Reply:
Done